marklogic-data-hub-central/ui/src/pages/Load.test.tsx

```diff
@@ -27,7 +25,19 @@ describe('Load component', () => {
         const authorityService = new AuthoritiesService();
         authorityService.setAuthorities(['readIngestion']);
 
-        const { getByText, getByTitle, getByLabelText, getByTestId, queryByTestId, queryByText, queryByTitle } = render(<AuthoritiesContext.Provider value={authorityService}><Load/></AuthoritiesContext.Provider>);
+        let getByText, getByTitle, getByLabelText, getByTestId, queryByTestId, queryByText, queryByTitle;
+        await act(async () => {
+          const renderResults = render(
```

Comment:
RTL commands are all internally wrapped in act.
const renderResults = await render(.......) will help get rid of the "wrap in act... " warning when running the test

Reply:
@wooldridge pushed small changes. 
- wrapping the component in act was sufficient to get rid of the warnings. 
- fixed unmount warning